static/app/components/layouts/thirds.tsx

```diff
@@ -111,7 +111,6 @@ export const HeaderNavTabs = styled(NavTabs)`
   }
   & > li > a {
     padding: ${space(1)} 0;
-    font-size: ${p => p.theme.fontSizeLarge};
```

Comment:
This property was added to reset the hard-coded value in `shared-components.less`. Since we're now removing that value, this is no longer necessary.